platform/darwin/test/MGLCodingTests.m

```diff
@@ -80,6 +80,32 @@ - (void)testPolyline {
     XCTAssertEqual([multiLine coordinate].latitude, multiLineCenter.latitude);
     XCTAssertEqual([multiLine coordinate].longitude, multiLineCenter.longitude);
     
+    CLLocationCoordinate2D segmentCoordinates[] = {
+        CLLocationCoordinate2DMake(35.040390, -85.311477),
+        CLLocationCoordinate2DMake(35.040390, -85.209510),
```

Comment:
This test case is for a straight line. I addressed testing a non linear polyline in a test case below.